src/Helper/OrganisationHelper.php

```diff
@@ -58,14 +57,14 @@ private function getSF1500(): SF1500 {
     if (NULL === $this->sf1500) {
 
       $soapClient = new SoapClient([
-        'cache_expiration_time' => explode(',', $this->settings->get(SettingsForm::CACHE_EXPIRATION)),
+        'cache_expiration_time' => explode(PHP_EOL, $this->settings->getCacheExpiration()),
```

Comment:
Wrapping the `explode` expression in in `array_filter(array_map('trim`, …))` is always a good idea.